### ISSUE TITLE
TMDb in TV Show Scrapper

### ISSUE THREAD

**Issue by rgponce** [Komet/MediaElch#894]
Hello, my native language is not English, so it is not always that TVDB contains all the information about TV shows in my language.

Would it be possible to add the option of choosing TMDb as a source of information for the scrapper fields of tv shows, in addition to TVDB and IMDB?

**Comment by bugwelle** [comment]
Hi there,

I would love to add TMDb as a TV show scraper but that requires quite some internal changes. It's on my to do list after #892 is finished. Adding another TV scraper is (at the moment) a lot of work because all parts of MediaElch rely on TheTvDb. I have to generalize the UI, etc.
It will take some time.

This issue is a duplicate of #712 :-)
